denonavr/audyssey.py

```diff
@@ -137,37 +156,93 @@ def _set_audyssey(self, parameter, value):
 
         return False
 
+
+class Audyssey(AppCommand0300):
+    """Audyssey Commands."""
+
+    def __init__(self, receiver):
+        """
+        Initialize Audyssey Settings of DenonAVR.
+
+        :param receiver: DenonAVR Receiver
+        :type receiver: DenonAVR
+        """
+
+        super().__init__(receiver)
+        self._init_commands(
+            set_cmd=AUDYSSEY_SET_CMD, get_cmd=AUDYSSEY_GET_CMD, params=AUDYSSEY_PARAMS
+        )
+        self.dynamiceq = None
+        self.multeq = None
+        self.reflevoffset = None
+        self.dynamicvol = None
+
     def dynamiceq_off(self):
         """Turn DynamicEQ off."""
-        if self._set_audyssey(parameter="dynamiceq", value=0) is True:
+        success = self._set(parameter="dynamiceq", value=0)
+        if success:
```

Comment:
We have a couple options for dealing with state getting out of sync here.  First, it probably makes the most sense to check if the receiver is off - it is off we can simply return and not update state.  Next, even if it is on and we receive successful reply, we may consider asking for a state update on the modified parameter before setting it.